aemcmc/conjugates.py

```diff
@@ -0,0 +1,64 @@
+import aesara.tensor as at
+from aesara.graph.kanren import KanrenRelationSub
+from aesara.graph.optdb import OptimizationDatabase
+from etuples import etuple, etuplize
+from kanren import eq, lall
+from unification import var
+
+conjugatesdb = OptimizationDatabase()
+
+
+def beta_binomial_conjugateo(observation_model_expr, value_expr, posterior_expr):
```

Comment:
Just like my previous comment, it's not clear why this relation should have three inputs instead of two.  We need our relations to directly represent equalities, so that they can be applied generally.